.travis.yml

```diff
@@ -3,14 +3,25 @@ matrix:
   include:
     - php: 5.6
     - php: 5.5
-# use modern infrastructure based on containers  
 sudo: false
 before_install:
   - cd ..
+  - wget --no-check-certificate https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.7.0.zip && unzip elasticsearch-1.7.0.zip
+  - mv elasticsearch-1.7.0 elasticsearch
+  - cd elasticsearch
+  - bin/elasticsearch -d
+  - cd ..
   - git clone https://github.com/neos/neos-base-distribution.git
   - cd neos-base-distribution
   - composer require flowpack/elasticsearch-contentrepositoryadaptor dev-master
 install:
   - composer install
+  - cd ..
+  - rm -rf neos-base-distribution/Packages/Application/Flowpack.ElasticSearch.ContentRepositoryAdaptor
+  - mv Flowpack.ElasticSearch.ContentRepositoryAdaptor neos-base-distribution/Packages/Application/Flowpack.ElasticSearch.ContentRepositoryAdaptor
+  - cd neos-base-distribution
+before_script:
+  - sleep 15
```

Comment:
I'll try some runs without the sleep. Well, the service will be started before but it might be possible that it is not up and running already.